client/state/data-layer/wpcom/read/tags/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_TAGS_REQUEST,
+} from 'state/action-types';
+import {
+	receiveTags,
+} from 'state/reader/tags/items/actions';
+
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+
+export function requestTags( store, action, next ) {
+	const path = action.payload && action.payload.slug
```

Comment:
this is a bit ambiguous - it really deserves some parenthesis to make the precedence ultra-clear

```js
const path = ( action.payload && action.payload.slug )
	? …
	: …
```